packs/salt/actions/lib/utils.py

```diff
@@ -0,0 +1,30 @@
+import yaml
+
+action_meta = {
+    "name": "",
+    "parameters": {
+        "action": {
+            "type": "string",
+            "immutable": True,
+            "default": ""
+        },
+        "kwargs": {
```

Comment:
Would This would lead to some opacity in salt action? I guess if the asnwer is checkout salt docs I guess that works.